sections/introduction.tex

```diff
@@ -1,10 +1,17 @@
 \section{Introduction}
 \label{sec:introduction}
 
-The Internet as a platform of communication has developed a one-and-only character in the previous decade: no matter where devices and users are located, they connect to the Internet and talk to each other. Even in scenarios where these devices are close to each other on the same network, local architecture is rarely leveraged and all traffic has to go through the global network, possibly traversing multiple autonomous systems throughout the globe. Driven in particular by the growing domain of the Internet of things (IoT) where different "smart" local devices talk to each other, a movement of Web applications towards making more use of local network infrastructure seems just as reasonable.
+ 
 
-Imagine a classroom with students following a presentation which is concluded with a vote from students' smart phones. Although students are connected to the same WiFi, such votes are usually performed using online voting platforms where all voters have to connect to the Internet for accessing the platform. Despite the obvious unnecessary traffic, this scenario has another significant problem: the voting cannot be performed upon failing Internet access. This seems almost abstruse knowing that all participants are connected to the same local network.
```

Comment:
This is a great example although it is an entire new paragraph that adds up to the intro content. Perhaps if we modify the first paragraph to be shorter we could keep it.